core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -2909,9 +2922,13 @@ private SqlDialect nonOrdinalDialect() {
         + "from \"foodmart\".\"reserve_employee\"";
     final String expected = "SELECT TRIM('$@*AABC$@*AADCAA$@*A', '$@*A')\n"
         + "FROM foodmart.reserve_employee";
+    final String expectedSqlite = "SELECT TRIM('$@*AABC$@*AADCAA$@*A', '$@*A')\n"
+        + "FROM \"foodmart\".\"reserve_employee\"";
     sql(query)
       .withBigQuery()
-      .ok(expected);
```

Comment:
Just write it like this?
```
.withBigQuery().ok(expected)
.withSqlite().ok(expectedSqlite);
```

Reply:
Done, thanks for your careful review. @xiedeyantu